src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,557 @@
+
+##### `font`
+
+The second most commonly preloaded resource is the `font`, which is a late-discovered resource since the browser will only download a font file after the layout phase when the browser knows that the font will be rendered on the page.
+
+##### `style`
+
+Stylesheets are ordinarily embedded in the document's `<head>` and discovered early during the document parsing. Additionally, as stylesheets are render-blocking resources they are assigned the _Highest_ request priority. This should make preloading stylesheets unnecessary but it is sometimes required to re-prioritize the requests. A <a hreflang="en" href="https://bugs.chromium.org/p/chromium/issues/detail?id=629420">bug</a> in Google Chrome (fixed in Chrome 95) prioritizes preloaded resources ahead of other higher-priority resources discovered by the preload scanner, including CSS files. Preloading the stylesheet will restore its _Highest_ priority. Another instance when stylesheets are preloaded is when they are not downloaded directly from the HTML document, such as the <a hreflang="en" href="https://www.filamentgroup.com/lab/async-css.html">asynchronous CSS</a> "hack" which uses an `onload` event to avoid render-blocking the page with non-critical CSS.
+
+##### `fetch`
+
+Preload may be used to initiate a request to retrieve data which you know is critical to the rendering of the page, such as a JSON response or stream.
+
+##### `image`
+
+Preloading images may help improve the LCP score when the image is not included in the initial HTML, such as a CSS `background-image`.
+
+#### The `crossorigin` attribute
+
+The `crossorigin` attribute is used to indicate whether CORS must be used when fetching the requested resource. This could apply to any resource type, but it is most commonly associated with font files as they should always be requested using CORS.
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>value</th>
+        <th>desktop</th>
+        <th>mobile</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>not set</td>
+        <td class="numeric">66.61%</td>
+        <td class="numeric">65.91%</td>
+      </tr>
+      <tr>
+        <td><code>crossorigin</code> (or equivalent)</td>
+        <td class="numeric">14.50%</td>
+        <td class="numeric">13.48%</td>
+      </tr>
+      <tr>
+        <td><code>use-credentials</code></td>
+        <td class="numeric">0.04%</td>
+        <td class="numeric">0.04%</td>
+      </tr>
+    </tbody>
+  </table>
+
+<figcaption>{{ figure_link(caption='`rel="preload"` `crossorigin` attribute values.', sheets_gid="1246058294", sql_file="hint_attribute_usage.sql") }}</figcaption>
```

Comment:
This isn't generating properly. Fill come back to it and investigate why.

Reply:
Fixed. Need to use `<code>` since this isn't markdown.

Reply:
Yeah it should support both though. But let's go with that for the moment.

Reply:
Ah it's cause it's in the `<figcaption>` tag within a `<table>` — unlike a graph embed which lives outside any HTML elements. OK that makes sense. Will still see if we can fix though as not totally obvious.

Reply:
I think if i use `<figure markdown>` as the opening element it would work, but since this is an HTML table I removed `markdown`

Reply:
Fixed this in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2458 . Up to you whether you wanna switch back to markup or happy to leave as HTML.